mscore/musescore.cpp

```diff
@@ -7247,8 +7247,8 @@ int main(int argc, char* av[])
             if (styleFile.isEmpty())
                   parser.showHelp(EXIT_FAILURE);
             }
-      useFactorySettings = parser.isSet("F");
-      deletePreferences = (useFactorySettings || parser.isSet("R"));
+      deletePreferences = parser.isSet("F");
+      useFactorySettings = (deletePreferences || parser.isSet("R"));
```

Comment:
Something is wrong with the naming.

Read above: `-F - Use factory settings`, `-R - Revert to factory settings, but keep default preferences`. And here you use `useFactorySettings` reading the `-R` parameter.

Why do you need to change these strings?

Reply:
-F deletes the settings files while -R does not. I simply used the same basic idea as the original code. If -F is used, then deletePreferences is set so useFactorySettings needs to be set. Prior to my changes, user settings were always deleted via the deletePreferences option if either -F or -R was used, so they actually did the exact same thing. 

If you look at the version 2 handbook under Command line options, you will see this is how -F and -R were documented to be implemented. I pointed this out in a forum post at some point. Rather than changing the code to make this happen, the handbook was changed instead.

In my testing of the PR it seems the results are what I expect, though I haven't used -R to fix any of the odd startup issues that seem to be mysteriously fixed by reverting to factory settings. This is because I haven't had the opportunity.